packages/cli/src/commands/base-command.ts

```diff
@@ -64,6 +64,9 @@ export abstract class BaseCommand extends Command {
 		await this.errorReporter.init({
 			serverType: this.instanceSettings.instanceType,
 			dsn: this.globalConfig.sentry.backendDsn,
+			environment: process.env.ENVIRONMENT ?? '',
+			release: process.env.N8N_VERSION ?? '',
+			serverName: process.env.DEPLOYMENT_NAME ?? '',
```

Comment:
should we add these to `packages/@n8n/config/src/configs/sentry.config.ts` ? I'd like us to reduce explicit `process.env` wherever possible.